### ISSUE TITLE
Custom properties not working with some elements

### ISSUE THREAD

**Issue by guilhermeocosta** [nuxodin/ie11CustomProperties#61]
** Description **  
After updating from v3 to v4, some elements is not receiving proper styles from the custom properties, but looking at the Developer Tools, the two elements of the example are receiving the values from custom properties in the same way, but for some reason it's not rendered properly.

Example 1: The big blue button "Próximo" is receiving 
<img width="1676" alt="Screen Shot 2020-07-09 at 6 20 37 PM" src="https://user-images.githubusercontent.com/22602055/87092249-7caa7580-c211-11ea-9b6f-bd273110148a.png">

Example 2: The tab should have a border-bottom in the same blue color as the button (I've outlined the element to be easier to find it in the UI)

<img width="1679" alt="Screen Shot 2020-07-09 at 6 21 40 PM" src="https://user-images.githubusercontent.com/22602055/87092327-9fd52500-c211-11ea-8e1d-d8b2c66dfec4.png">

** Version **  
`4.1.0`

Any guesses on this? What should I do to test it better?

Thanks in advance

**Comment by nuxodin** [comment]
Thanks for your feedback.
Can it be that it always concerns !important?

**Comment by guilhermeocosta** [comment]
I do use [postcss-important](https://github.com/morishitter/postcss-important) because of the nature of this specific application. I will try to disable it to test, thanks for the quick answer.

**Comment by guilhermeocosta** [comment]
Yes, the `!important` was messing with the specificity. Is there a way to the applied values respects the `!important` flag? It's stripping it away in the execution

**Comment by coliff** [comment]
heya @guilhermeocosta can you please test the proposed fix in #63 (files)

If you could post back here with any feedback on that it'd be appreciated!
